ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/InsertValuesExecutor.java

```diff
@@ -206,30 +206,30 @@ private void validateInsert(final List<ColumnName> columns, final DataSource dat
     final List<String> headerColumns;
     if (columns.isEmpty()) {
       headerColumns = dataSource.getSchema().headers()
-          .stream()
-          .map(column -> column.name().text())
-          .collect(Collectors.toList());
+              .stream()
```

Comment:
There are several changes that has only empty spaces. Could you change the IDE to prevent adding these spaces automatically?

Reply:
+1, small things like this go a long way in terms of making diffs easier to review. Also keeps the git blame history cleaner, in case someone needs to go back and see when certain lines/files were last modified.